<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1065</width>
    <height>648</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Process count:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Matrix size:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Row count:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Column count:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_column_count">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>10000</number>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_row_count">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>120</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1000</number>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_killall">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>291</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Kill all</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_kill">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>410</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Kill</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_suspend">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Suspend</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_resume">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>510</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Resume</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Process:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>560</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Change priority:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_priority">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>560</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <item>
     <property name="text">
      <string>Real time</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>High</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Above normal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Normal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Below normal</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Idle</string>
     </property>
    </item>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>581</width>
      <height>561</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="rowCount">
     <number>1</number>
    </property>
    <property name="columnCount">
     <number>8</number>
    </property>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>144</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>144</number>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>65</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>65</number>
    </attribute>
    <row/>
    <column>
     <property name="text">
      <string>Pid</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Status</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Priority</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Time</string>
     </property>
    </column>
    <column/>
    <column/>
    <column/>
    <column/>
   </widget>
   <widget class="QComboBox" name="comboBox_process_count">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
